doc/antora/modules/ROOT/pages/faq.adoc

```diff
@@ -213,7 +213,7 @@ User entry in raddb/users file example
 .CISCO configuation example
 [%collapsible]
 ====
-	aaa authorization network default radius
```

Comment:
this is an example of configuration from an external product.  we can't change that :(